services/orderService.js

```diff
@@ -0,0 +1,20 @@
+const orderDao = require("../models/orderDao");
+
+const modifyOrderCount = async (foodId, quantity, userId) => {
+        return await orderDao.modifyOrderCount(foodId, quantity, userId);
+};
+
+const deleteOrder = async(deleteOrderItem, userId) =>{
+    try {
+        for(let i = 0; i <= deleteOrderItem.length - 1; i++) {
+            let food_id = deleteOrderItem[i].foodId;
+            await orderDao.deleteOrderItems(food_id, userId);
+        }
+    } catch (err) {
+        const error = new Error("COULD NOT DELETE DATA");
+        error.statusCode = 404;
+        throw error;
+    }
+};
```

Comment:
```suggestion
const deleteOrder = async (deleteOrderItem, userId) => {
  try {
    await Promise.all(
      deleteOrderItem.map(async (item) => {
        const { foodId } = item;
        await orderDao.deleteOrderItems(foodId, userId);
      })
    );
  } catch (error) {
    throw new Error("COULD NOT DELETE DATA");
  }
};
```


- 위 내용으로 수정하여, for문이 도는 행위가 끝났을 때 해당 데이터를 전달하여 db를 건드리는 빈도수를 최소화 해주세요. 해당 내용을 가능케하는 Promise.all() 메소드의 기능을 공식문서를 통해서 확인해주세요!
- 또한 map 함수를 돌려서 가독성을 확보해주세요!